### PR TITLE
[STAKAN-346] Terms and conditions of Moonbeam crowdloan

### PR DESCRIPTION
**Pull request by BigBadAlien**
Hello, brave developer, nice day to merge a pull request, ain't it?

## Follow the checklist to proceed

- [x] I've performed a code self-review
- [x] I've tested feature or bugfix on my own
- [x] If I make changes to another person's module, I've asked how to use it or request a review
- [x] I've checked mobile & tablet screens (if have changes in UI)
- [x] Local build successfully completed
- [x] Branch name and pull request title are according to [the requirements](https://ankrnetwork.atlassian.net/wiki/spaces/STAKAN/pages/1666646162/Branches+and+pull+request+naming)

### CODE REVIEW

File: src/UiKit/Checkbox/CheckboxField.tsx [hunk 1]
```diff
@@ -11,13 +11,9 @@ interface ISwitchFieldProps extends FieldRenderProps<HTMLElement> {
...
-export const CheckboxField = ({
-  label,
-  input,
-  meta,
-  children,
-}: ISwitchFieldProps & any) => {
-  const classes = useCheckboxStyles();
+export const CheckboxField = (props: ISwitchFieldProps & any) => {
+  const { label, input, meta, children } = props;
+  const classes = useCheckboxStyles(props);
```
Comment (Antroll) [inline src/UiKit/Checkbox/CheckboxField.tsx:16]: Look like there is no reason to do that.

Comment (BigBadAlien) [reply src/UiKit/Checkbox/CheckboxField.tsx:16]: I added it to override class here https://app.circleci.com/pipelines/github/Ankr-network/stkr-web3/2639/workflows/ac318984-22f7-464c-b86e-068ef25eba00

**Review by Antroll — APPROVED** [review]

File: src/UiKit/Checkbox/CheckboxField.tsx (hunk 1 above)
Comment (BigBadAlien) [reply src/UiKit/Checkbox/CheckboxField.tsx:16]: Otherwise

```
                classes={{ checkbox: classes.checkbox }}
```

wouldn't work

**Comment by BigBadAlien** [comment]
While we were implementing moonbeam terms and conditions their crowdloan completed

https://moonbeam.foundation/moonbeam-crowdloan/

Screenshot 2021-11-22 at 21 05 51

**Pull request closed without merging**